internal/command/deploy/machines.go

```diff
@@ -116,6 +116,15 @@ type machineDeployment struct {
 	deployRetries         int
 }
 
+type PostDeploymentInfo struct {
+	FlyctlVersion string `json:"flyctl_version"`
+	Error         string `json:"error"`
+}
+
+type ReleaseMetadata struct {
+	PostDeploymentInfo PostDeploymentInfo `json:"post_deployment_info"`
+}
```

Comment:
We need to delete these in favour of https://github.com/superfly/fly-go/pull/88 
```suggestion
```

Once that is merged, we need to bump the `fly-go` version in flyctl and then use that type.